app/utils/results.py

```diff
@@ -222,3 +223,41 @@ def add_ip_card(html_soup: BeautifulSoup, ip: str) -> BeautifulSoup:
         # Inserting the element
         ref_element.insert_before(ip_tag)
     return html_soup
+
+
+def get_tabs_content(tabs: dict,
+                     full_query: str,
+                     search_type: str,
+                     translation: dict) -> dict:
+    """Takes the default tabs content and updates it according to the query.
+
+    Args:
+        tabs: The default content for the tabs
+        full_query: The original search query
+        search_type: The current search_type
+        translation: The translation to get the names of the tabs
+
+    Returns:
+        dict: contains the name, the href and if the tab is selected or not
+    """
+    tabs = copy.deepcopy(tabs)
+    for tab_id, tab_content in tabs.items():
+        # update name to desired language
+        if tab_id in translation:
+            tab_content['name'] = translation[tab_id]
+
+        # update href with query
+        query = full_query
+        if '&tbm=' in full_query:
+            query = full_query.replace(f"&tbm={search_type}", '')
```

Comment:
Can be simplified to

```suggestion
        query = full_query.replace(f'&tbm={search_type}', '')
```

Reply:
Good point!